### PR TITLE
Update github repo urls for gitian and building

### PR DESCRIPTION
**Pull request by schancel**
As per title.

### CODE REVIEW

File: web/docs/docusaurus.config.js [hunk 1]
```diff
@@ -1,93 +1,93 @@
 module.exports = {
```
Comment (EyeOfPython) [inline web/docs/docusaurus.config.js:1]: ?

File: src/init.cpp [hunk 2]
```diff
@@ -2020,13 +2018,12 @@ bool AppInitParameterInteraction(Config &config, const ArgsManager &args) {
     }
 
     fRequireStandardPolicy = !args.GetBoolArg("-acceptnonstdtxn", false);
-    fRequireStandardConsensus =  // defaults to fRequireStandardPolicy
+    fRequireStandardConsensus = // defaults to fRequireStandardPolicy
         !args.GetBoolArg("-allownonstdtxnconsensus", !fRequireStandardPolicy);
     if (!fRequireStandardPolicy && fRequireStandardConsensus) {
-        return InitError(
```
Comment (EyeOfPython) [inline src/init.cpp:2026]: Let's do formatting in a separate commit

**Review by EyeOfPython — CHANGES_REQUESTED** [review]

File: web/docs/docusaurus.config.js (hunk 1 above)
Comment (schancel) [reply web/docs/docusaurus.config.js:1]: Yeah, I'll go back through and remove formatting changes

File: src/test/rpc_tests.cpp [hunk 3]
```diff
@@ -158,7 +158,7 @@ BOOST_AUTO_TEST_CASE(rpc_rawsign_missing_amount) {
     // Old format, missing amount parameter for prevout should generate
     // an RPC error.  This is because of new replay-protected tx's require
     // nonzero amount present in signed tx.
-    // See: https://github.com/Bitcoin-ABC/bitcoin-abc/issues/63
+    // See: https://github.com/LogosFoundation/lotusd/issues/63
```
Comment (schancel) [inline src/test/rpc_tests.cpp:161]: This shouldn't have changed

File: src/CMakeLists.txt [hunk 4]
```diff
@@ -49,7 +49,7 @@ if(NOT CMAKE_SYSTEM_NAME IN_LIST OS_WITH_JEMALLOC_AS_SYSTEM_DEFAULT)
 endif()
 
 # FIXME: Building against jemalloc causes the software to segfault on OSX.
-# See https://github.com/Bitcoin-ABC/bitcoin-abc/issues/401
+# See https://github.com/LogosFoundation/lotusd/issues/401
```
Comment (EyeOfPython) [inline src/CMakeLists.txt:52]: Undo

**Review by EyeOfPython — CHANGES_REQUESTED** [review]

File: contrib/debian/copyright [hunk 5]
```diff
@@ -1,7 +1,7 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: Bitcoin ABC
```
Comment (EyeOfPython) [inline contrib/debian/copyright:2]: Should this be changed too?

**Review by EyeOfPython — APPROVED** [review]
